README.md

```diff
@@ -107,6 +107,12 @@ Support this project with your organization. Your logo will show up here with a
 <a href="https://opencollective.com/react-toastify/organization/8/website"><img src="https://opencollective.com/react-toastify/organization/8/avatar.svg"></a>
 <a href="https://opencollective.com/react-toastify/organization/9/website"><img src="https://opencollective.com/react-toastify/organization/9/avatar.svg"></a>
 
+### ToastContainer Props
```

Comment:
I don't think this should be added here, the README seems to contain only high-level information about the project and not detailed release notes.